test/intl402/DurationFormat/prototype/format/en-us-style-options.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.DurationFormat.prototype.format
+description: Checks the behavior of Intl.DurationFormat.prototype.format() in English.
+features: [Intl.DurationFormat]
+locale: [en-US]
+---*/
+
+const stylesList = ["long" , "short" , "narrow", "digital"];
+
+const assertList = {
+  "PT1H50M":  {
+    "long": "1 hour and 50 minutes",
+    "short": "1 hr, 50 min",
+    "narrow": "1h 50m",
```

Comment:
Not sure if `hr` or `hr.`

Reply:
Do we normally put the localized strings in the Test262 suite? Adding and removing `.` is a common thing that changes across CLDR updates

Reply:
There are some cases at relative time format : 

https://github.com/tc39/test262/blob/main/test/intl402/RelativeTimeFormat/prototype/format/en-us-numeric-auto.js#L29 

I'm happy to only check the resolvedOptions of those methods instead avoiding depending on CLDR

Reply:
I was asking about the general policy of Test262 and CLDR-dependent output. I think it's useful to have at least a few tests for the expected output of the format function, but we just need to be aware that these tests are data-dependent, and the expectations could drift.

Reply:
This is what the `locale` key in the front matter is for